### PR TITLE
Updating diagrams with new styles

### PR DESCRIPTION
**Pull request by hillarymc**
Updated the following diagrams to follow similar and cohesive styling

- otel general architecture diagram
- 2 otel baggage diagrams
- traces venn diagram 
- tail sampling process image
- waterfall trace image
- otel gateway lbs sdk v2 image

---

- Contributes to #780

### CODE REVIEW

File: static/img/otel_baggage_v2.svg
Comment (svrnm) [inline static/img/otel_baggage_v2.svg:1]: Rename to otel_bagage_2.svg

**Review by svrnm — DISMISSED** [review]
overall looks good to me, only nit is that you have some _v2 suffixes that I think we can remove.

File: content/en/docs/collector/deployment/gateway.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ same collector instance where the tail sampling policy is applied.
 
 Let's have a look at such a case where we are using the load-balancing exporter:
 
-![Gateway deployment with load-balancing exporter](../../img/gateway-lb-sdk.svg)
+![Gateway deployment with load-balancing exporter](../../img/otel_gateway_lb_sdk_v2.svg)
```
Comment (chalin) [inline content/en/docs/collector/deployment/gateway.md:31]: Per the following issue, please keep the kebab case in the filename:

- #780

Yeah, I know that the file names of all of the other files that you've updated the images for use snake case 🤷🏼‍♂️, but we're actually try to aim for kebab case as the standard.

(No need to change any of the other file names to kebab case in this PR, but if you do, that would be grand.)

**Review by chalin — APPROVED** [review]
Thanks for renaming the files @hillarymc, and for contributing the SVGs, of course!

LGTM, though I haven't cross-checked the diagrams themselves. I'll let @svrnm and @cartermp validate the diagram contents.

**Review by cartermp — APPROVED** [review]
Extremely into these updates. Thanks @hillarymc!

**Pull request merged**